controllers/drain_controller.go

```diff
@@ -172,6 +172,29 @@ func (dr *DrainReconcile) Reconcile(ctx context.Context, req ctrl.Request) (ctrl
 				return reconcile.Result{RequeueAfter: 5 * time.Second}, nil
 			}
 
+			// check the device plugin exited and enable it again
+			// only of we have something in the node state spec
+			if len(nodeNetworkState.Spec.Interfaces) > 0 {
+				completed, err = dr.enableSriovDevicePlugin(ctx, node)
```

Comment:
See my other comment, completed is a confusing name here unless we change the implementation of `enableSriovDevicePlugin`.

Reply:
done